core/constants.py

```diff
@@ -121,7 +122,16 @@
 
 TABLES = ["core_history", "django_admin_log"]
 
-NOTIFICATION_TITLES = ()
+# Notification titles
+INCIDENT_REPORT_CREATED_NOTIFICATION_TITLE = "New Incident Report"
+INCIDENT_REPORT_STATUS_UPDATED_NOTIFICATION_TITLE = "Incident Report Status Update"
+# Notification Bodies
+INCIDENT_REPORT_CREATED_NOTIFICATION_BODY = Template(
+    "$email submitted a new incident report {$link}."
```

Comment:
Are the curly braces intentional seeing that they would display in the resulting string?

Reply:
@kmazi Thanks for the noting the mistake. I have removed them